docs/clientapi.md

```diff
@@ -22,6 +22,11 @@ new Irc.Client({
         username: '*',
         hostname: 'users.host.isp.net',
         ip: '1.1.1.1',
+        options: {
+            secure: '',
```

Comment:
`secure` should be a boolean so that its tag ends up as just `secure` without a value